champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/mage/axe/LightningOrb.java

```diff
@@ -113,43 +107,44 @@ public SkillType getType() {
-
-                if (count >= maxTargets) continue;
-                throwableItem.getImmunes().add(ent);
-                if (ent instanceof Player target) {
-                    championsManager.getEffects().addEffect(target, EffectTypes.SHOCK, (long) (getShockDuration(level) * 1000));
-                }
-
-                championsManager.getEffects().addEffect(ent, playerThrower, EffectTypes.SLOWNESS, slowStrength, (long) (getSlowDuration(level) * 1000));
-
-                playerThrower.getLocation().getWorld().strikeLightning(ent.getLocation());
-                UtilDamage.doCustomDamage(new CustomDamageEvent(ent, playerThrower, null, DamageCause.CUSTOM, getDamage(level), false, getName()));
-
-                //ent.getWorld().playSound(ent.getLocation(), Sound.ENTITY_LIGHTNING_BOLT_THUNDER, 1.0F, 1.0F);
-                count++;
-            }
+            activateOrb(playerThrower, throwableItem, level);
         }
 
         throwableItem.getItem().remove();
     }
 
     @Override
     public void onTick(ThrowableItem throwableItem) {
-        throwableItem.getLastLocation().getWorld().playSound(throwableItem.getLastLocation(), Sound.BLOCK_LAVA_EXTINGUISH, 0.6f, 1.6f);
-        throwableItem.getLastLocation().getWorld().spawnParticle(Particle.FIREWORK, throwableItem.getLastLocation(), 1);
+        if ((throwableItem.getAge() / 50) > getDelay(getLevel((Player) throwableItem.getThrower())) * 20) {
+            activateOrb((Player) throwableItem.getThrower(), throwableItem, getLevel((Player) throwableItem.getThrower()));
+            System.out.println("removed here");
+            System.out.println("age" + throwableItem.getAge());
+            System.out.println("required age: " + getDelay(getLevel((Player) throwableItem.getThrower())) * 20);
+            throwableItem.getItem().remove();
+        } else {
+            throwableItem.getLastLocation().getWorld().playSound(throwableItem.getLastLocation(), Sound.BLOCK_LAVA_EXTINGUISH, 0.6f, 1.6f);
+            throwableItem.getLastLocation().getWorld().spawnParticle(Particle.FIREWORK, throwableItem.getLastLocation(), 1);
+        }
+    }
+
+    private void activateOrb(Player playerThrower, ThrowableItem throwableItem, int level) {
+        for (LivingEntity ent : UtilEntity.getNearbyEnemies(playerThrower, throwableItem.getItem().getLocation(), getRadius(level))) {
+            if (!throwableItem.getImmunes().contains(ent)) {
+                championsManager.getEffects().addEffect(ent, playerThrower, EffectTypes.SLOWNESS, slowStrength, (long) (getSlowDuration(level) * 1000));
+                championsManager.getEffects().addEffect(ent, EffectTypes.SHOCK, (long) (getShockDuration(level) * 1000));
+                playerThrower.getLocation().getWorld().strikeLightning(ent.getLocation());
+                UtilDamage.doCustomDamage(new CustomDamageEvent(ent, playerThrower, null, DamageCause.CUSTOM, getDamage(level), false, getName()));
+            }
```

Comment:
Do we want a line of sight check here? Feel like just throwing it next to someones walled in base could be really annoying

Reply:
oh yeah I was gonna add that I forgot thank you